### ISSUE TITLE
What's the 'strain field' parameter means in inclusiongen

### ISSUE THREAD

**Issue by SlowtyperChen** [LLNL/ParaDiS#8]
> In ParaDiS precipitates are represented as Eshelby inclusions. To activate this feature, first make sure the code is compiled with option `-DESHELBY` (line 336 of `makefile.setup`). Then you can take a look at example `tests/onescrew.ctrl`. In this control file you can see that the precipitate/inclusion file is specified with option `inclusionFile`, e.g. `inclusionFile = "tests/onescrew.dat"`. In this example, the precipitates configuration is defined in file `tests/onescrew.dat`, where each line specifies a precipitate using the following format:
> ```
> #
> # Data for an precipitate consists of the following items separated with white space
> #
> # Precipitate ID     : 1 integer identifying the precipitate regardless of its position in the simulation or the domain encompassing it. (Should be sequential values)
> # Position           : 3 values specifying the XYZ coordinates of the center of the precipitate
> # Semi-principal axes: 3 values defining the three semi-principal axis of the ellipsoidal precipitate in units of b
> # Rotation matrix    : 2 vectors defining the rotation of the ellipsoidal precipitates
> # Strain field       : 6 components of the strain field for the particleStrain field is a symmetric matrix so only six components are specified
> #
> ```
> - If you want to place precipitates at very specific locations then you can generate the precipitate file manually following the format above.
> - For randomly positioned spherical inclusions you can use tool `bin/inclusiongen`
> - For randomly positioned more specific types of inclusions (e.g. ellispoid T or ThetaPrime precipitates) you can use tool `bin/precipitates` 

 _Originally posted by @nrbertin in [#7](https://github.com/LLNL/ParaDiS/issues/7#issuecomment-2552512396)_

I have used the precipitates module for calculating DDD simulations containing inclusions. But I don't really understand what the parameter 'strain field' means here.

Is it the eig-strain in Eshelby's theory of inclusions?

**Comment by nrbertin** [comment]
Correct, it is the eigenstrain to compute the stress field associated with the inclusion.

**Comment by SlowtyperChen** [comment]
I see, thank you！

**Comment by SlowtyperChen** [comment]
@nrbertin Pro. nrbertin, I found that here is no option about the **Elastic Parameters** (shear modulus or Poisson rate) when using `inclusiongen` or `precipitation`. But according to Eshelby's theory, the stress field within the inclusion is rely on the Poisson rate and shear modulus of precipitations. Does the precipitate module of ParaDiS only consider the stress field outside of the inclusion?

**Comment by nrbertin** [comment]
You can use control parameters `shearModulus2` and `pois2` to define the elastic parameters associated with the inclusions. To compute forces inside of the inclusions one needs to compile the code with option `-DESHELBYFORCE` (line 342 of `makefile.setup`).
